presto-main/src/main/java/com/facebook/presto/metadata/FunctionAndTypeManager.java

```diff
@@ -394,7 +394,7 @@ public List<Type> getTypes()
         return builtInTypeAndFunctionNamespaceManager.getTypes();
     }
 
-    public Collection<ParametricType> getParametricTypes()
+    public List<ParametricType> getParametricTypes()
```

Comment:
Needs javadoc specifying the return value is immutable since that is *not* included in the type signature.